libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFSDKOAuthClient.m

```diff
@@ -55,7 +55,26 @@
 static NSString * const kSFRevokePath = @"/services/oauth2/revoke";
 
 static Class<SFSDKOAuthClientProvider> _clientProvider = nil;
+@interface UINavigationController(SFSDKOAuth)
+@end
+
+@implementation UINavigationController(SFSDKOAuth)
```

Comment:
allow us to delegate the call to the  login view controller subclass if required.